README.md

```diff
@@ -1,17 +1,92 @@
-# Vega API auto-generated gRPC clients
+# Vega API
 
 Version: 0.33.0
 
-This repository contains auto-generated gRPC clients for various languages.
+This repository contains everything you need for interacting with the Vega APIs.
+
+There are currently 3 APIs:
+* gRPC
+* GraphQL
+* REST
+
+## gRPC
+
+See https://grpc.io/ for general information on gRPC.
+
+Vega provides a rich gRPC API. For a complete list of endpoints, see [`proto/api/trading.proto`](https://github.com/vegaprotocol/api/blob/develop/proto/api/trading.proto):
+
+* Use `TradingService` for:
+  * `Prepare*` endpoints are there for convenience, and will be removed in the near future
+  * `SubmitTransaction` receives and executes a signed transaction
+* Use `TradingDataService` for public information on:
+  * assets, e.g. Ethereum ERC20 tokens
+  * balances for accounts: general, margin
+  * market data and metadata
+  * information on orders and trades
+  * Governance data: proposals, votes
+  * event stream subscriptions for the above
+
+Before writing your own API client (either manually or using auto-generation), check if a pre-created on is already available. See [Auto-generated gRPC clients](#auto-generated-grpc-clients) below.
+
+## GraphQL
+
+See https://graphql.org/ for general information on GraphQL.
+
+Vega provides a fully-featured GraphQL API. For the complete schema, see [`graphql/schema.graphql`](https://github.com/vegaprotocol/api/blob/develop/graphql/schema.graphql).
+
+The GraphQL Playground is enabled for convenience on Testnet non-validator nodes: https://lb.testnet.vega.xyz/playground.
+
+It is possible to use nested queries that return only the desired information, for example:
```

Comment:
**thought (non-blocking)**: This section is selling GraphQL as a concept rather than the APIs. I think the link above is sufficient, so I’d drop the example below. A nice upside of that is we don’t have to maintain the example.

Reply:
ok, I'll remove the example.

Reply:
Removed in 8c57df7.